tests/Fixer/Phpdoc/PhpdocScalarBreakdownFixerTest.php

```diff
@@ -0,0 +1,293 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Phpdoc;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Andreas Frömer <blubb0r05+github@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\AbstractPhpdocTypesFixer
+ * @covers \PhpCsFixer\Fixer\Phpdoc\PhpdocScalarBreakdownFixer
+ */
+final class PhpdocScalarBreakdownFixerTest extends AbstractFixerTestCase
+{
+    public function testBasicFix()
+    {
+        $expected = <<<'EOF'
+<?php
+    /**
+     * @return int|string|float|bool
```

Comment:
I would suggest to turn that into `string|int|float|bool`, to group numbers together

Reply:
let us use `bool|float|int|string` so we have numbers grouped and alpha order ;)